packages/data-types/__tests__/data-types.js

```diff
@@ -1,9 +1,144 @@
-import { encodeMainframeID, isValidMainframeID } from '..'
+import multibase from 'multibase'
+
+import { isValidMFID, MFIDParts, MFIDData, MFID } from '..'
 
 describe('data-types', () => {
-  it('encodes and validates Mainframe IDs', () => {
-    expect(isValidMainframeID('test')).toBe(false)
-    const id = encodeMainframeID(Buffer.from('test'))
-    expect(isValidMainframeID(id)).toBe(true)
+  const mbencoded = multibase.encode('base64url', 'test')
+
+  it('isValidMFID() checks if a string is a valid MFID', () => {
+    const id = MFID.create('user:test', 'utf8:bob').toString()
+    expect(isValidMFID(id)).toBe(true)
```

Comment:
I'm probably missing something around this test code, but how does a prefix of "user:" or "utf8:" pass validation here? Aren't MFIDs required to have an "mf:" prefix?

Reply:
Yes that's what `MFID.create()` does: https://github.com/MainframeHQ/js-mainframe/pull/145/files#diff-946a7304e72552038e78222f05e2e8b9R116